.github/workflows/ci.yml

```diff
@@ -125,10 +125,45 @@ jobs:
       - name: Run Front-end Tests
         run: |
           blt tests:frontend:run --define drush.alias='${drush.aliases.ci}' --define tests.run-server=false --environment=ci --ansi --verbose --no-interaction
+      - name: Install cypress
+        run: |
+              blt tests:cypress:npm-install --define drush.alias='${drush.aliases.ci}' --environment=ci --ansi --verbose --no-interaction
```

Comment:
What happens when you run `blt tests:all`? Do these get run everytime? Just wondering if npm-install should be outside of the tests namespace.

Reply:
The steps have been consolidated to expect "pre-tests" to be the first in the list.

Additionally, the logic has been updated so `npm ci` is only run if the node modules have not been previously installed.